### DISCUSSION TITLE
Loading static assets

### DISCUSSION THREAD

**Discussion by vimkaf** [leafsphp/leaf#182] [Q&A] [answered]
Hey Leaf Ninjas,

I'm stuck ...how does leaf load statics assets like css and js files...I have tried appending the base url to the assets path but i am getting the resource not found error.

`http://localhost:5500/assets/vendor/aos/dist/aos.css`

Is there something i'm overlooking??

**Comment by mychidarko** [comment, marked as answer]
Hi @vimkaf, you'll need to give a little more context about your setup

**Reply by vimkaf** [reply]
Just a regular normal setup served through `php leaf serve`

**Reply by mychidarko** [reply]
You mean Leaf MVC?

**Reply by mychidarko** [reply]
Can you show a snippet of how you're trying to load the CSS file?

**Reply by vimkaf** [reply]
Ok i have my css,js,images files in `public/assets`. so in the index.blade.php view file i have something like this

`<link href="/assets/vendor/simplebar/dist/simplebar.min.css" rel="stylesheet" type="text/css" media="all">`

but i am getting an error that the resource could not be found

Even if i do something like

`<link href="http://localhost:5500/assets/vendor/simplebar/dist/simplebar.min.css" rel="stylesheet" type="text/css" media="all">`

it's not working except i do

`<link href="http://localhost:5500/public/assets/vendor/simplebar/dist/simplebar.min.css" rel="stylesheet" type="text/css" media="all">`

So I want to know what's the ideal way to reference static assets in leafphp inside the view files. I was expecting something like a function `baseUrl("....")` or `app()->baseUrl("....")`.

**Reply by mychidarko** [reply]
Try this:

```php
<link href="<?php echo PublicPath('assets/vendor/simplebar/dist/simplebar.min.css'); ?>" rel="stylesheet" type="text/css" media="all">
```
